### PR TITLE
Feature/national context merge charts

### PR DESCRIPTION
**Pull request by weberjavi**
This PR updates Country Context > Socioeconomic indicators > Population chart
[Pivotal](https://www.pivotaltracker.com/story/show/164163348)
Economic Profile chart will be updated in another PR
kapture 2019-02-27 at 13 51 13

### CODE REVIEW

**Comment by simaob** [comment]
There are some errors/warnings on the inspector, but other than that it looks good to me. =)

**Comment by kfilip94** [comment]
Few small things, nothing critical:
1. It would be better to replace `NaN` value with `n/a` string for `null / undefined` values:
screenshot from 2019-02-27 15 05 57

2. For the `Population density` maybe you could change the value format because we have values like 903.0 and 2.034k which is basically 2034, it looks a bit weird
screenshot from 2019-02-27 15 08 37

Also, to be more consistent with our other charts, it would be nice to sort all values on the tooltip in descending order, but only if it is easy to do.

**Comment by weberjavi** [comment]
@kfilip94 thanks for the comments, working on it

**Comment by weberjavi** [comment]
@kfilip94 ready again ☝️

**Comment by simaob** [comment]
Found a small issue, I don't seem to be able to remove "India" from my selection, even though it has an `x` next to it.

File: app/javascript/app/pages/country-context/socioeconomic-indicators/population/population-selectors.js [hunk 1]
```diff
@@ -217,12 +222,13 @@ const getBarChartData = createSelector(
...
+            indicator: unitLabels[selectedIndicatorValue]
+              ? unitLabels[selectedIndicatorValue]
+              : 'People',
             theme: getThemeConfig(getYColumn(chartData)),
-            labelFunction: unitLabels[unit] ? unitLabels[unit] : 'People',
-            formatFunction: unitLabels[unit]
+            formatFunction: formatY[unit]
               ? getCustomYLabelFormat(unit)
-              : value => `${format(',.4s')(`${value}`).replace('G', 'B')}`
+              : value => `cona${format(',.4s')(`${value}`).replace('G', 'B')}`
```
Comment (kfilip94) [inline app/javascript/app/pages/country-context/socioeconomic-indicators/population/population-selectors.js]: @weberjavi What does 'cona' mean here?

**Comment by kfilip94** [comment]
@weberjavi Thank you for changes! :+1:  Could you change string `No Data` to `n/a`? We are using this placeholder on the flagship platform, it'll be more consistent I think:
screenshot from 2019-02-27 18 06 58
Overall, great! :1st_place_medal:

**Comment by weberjavi** [comment]
Now India can be deleted (is the default data on first load) and added a colour cache (as in some other charts)
@simaob , @kfilip94 ☝️

File: app/javascript/app/pages/country-context/socioeconomic-indicators/population/population-selectors.js (hunk 1 above)
Comment (weberjavi) [reply app/javascript/app/pages/country-context/socioeconomic-indicators/population/population-selectors.js]: 🤦‍♂️

**Comment by weberjavi** [comment]
Here you go 💹

**Pull request merged**